src/commands/wizard/authorizeCommand.ts

```diff
@@ -24,7 +24,7 @@ export class AuthorizeCommand {
             if (!result || result.title === l10n.t('No')) {
                 return Promise.resolve(false);
             } else {
-                await commands.executeCommand('sfdx.force.auth.web.login');
+                await commands.executeCommand('sfdx.org.login.web');
```

Comment:
This is actually broken in the current code. 😔 Apparently the command names moved on and we didn't know it.